metapackages/auto-instrumentations-node/package.json

```
@@ -0,0 +1,58 @@
+{
+  "name": "@opentelemetry/auto-instrumentations-node",
+  "version": "0.14.0",
+  "description": "Metapackage which bundles opentelemetry node core and contrib instrumentations",
+  "author": "OpenTelemetry Authors",
+  "homepage": "https://github.com/open-telemetry/opentelemetry-js#readme",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "repository": "open-telemetry/opentelemetry-js-contrib",
+  "scripts": {
+    "clean": "rimraf build/*",
+    "codecov": "nyc report --reporter=json && codecov -f coverage/*.json -p ../../",
+    "compile": "tsc -p .",
+    "lint": "eslint . --ext .ts",
+    "lint:fix": "eslint . --ext .ts --fix",
+    "precompile": "tsc --version",
+    "prepare": "npm run compile",
+    "tdd": "yarn test -- --watch-extensions ts --watch",
+    "test": "nyc ts-mocha -p tsconfig.json 'test/**/*.ts'",
+    "watch": "tsc -w"
+  },
+  "bugs": {
+    "url": "https://github.com/open-telemetry/opentelemetry-js-contrib/issues"
+  },
+  "devDependencies": {
+    "@types/node": "14.0.27",
+    "@types/mocha": "7.0.2",
+    "@types/sinon": "9.0.11",
+    "codecov": "3.7.2",
+    "gts": "3.1.0",
+    "mocha": "7.2.0",
+    "nyc": "15.1.0",
+    "rimraf": "3.0.2",
+    "sinon": "9.2.3",
+    "ts-mocha": "8.0.0",
+    "ts-node": "9.0.0",
+    "tslint-consistent-codestyle": "1.16.0",
+    "tslint-microsoft-contrib": "6.2.0",
+    "typescript": "4.1.3"
+  },
+  "dependencies": {
+    "@opentelemetry/api": "^0.18.0",
```

0.18.1 is latest, but rc.0 will be latest pretty soon. We should probably hold off rc.0 until we can update the whole repo at once

```suggestion
    "@opentelemetry/api": "^0.18.1",
```

---

this should be updated once for all repo before release, there are already other PRS that are merged pointing to `0.18.0`